### ISSUE TITLE
Accessibility: MAS20: alt text  is empty for the Rating images.

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#1]
#A11yMAS
#MAS20
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0

**Repro Steps:**
1.Open https://marketplace.visualstudio.com/VSCode
2..Press F12 key and check the alt text for  the "Rating" images below the icons under "Featured ","Most Popular ","Trending ","Recently Added " sections of the page .

Note:Same issue is observed in below link
https://marketplace.visualstudio.com/items?itemName=donjayamanne.python (Observe the rating stars in the page)

**Actual Result:**
The alt text is empty for the "Rating" images   below the icons under "Featured ","Most Popular ","Trending ","Recently Added " sections of the page .

**Expected Result:**
The alt text should be mentioned for "Rating" images below the icons under "Featured ","Most Popular ","Trending ","Recently Added " sections of the page .
Note: As the alt text is empty , the narrator reads the rating stars as "image"

**User Impact:**
Visually Impaired users relying on screen readers gets confused if the its is just announcing as image instead of "Rating stars" and may not use the application efficiently.
![alt text is empty for ratings](https://cloud.githubusercontent.com/assets/24825441/21671549/24cdd508-d342-11e6-8144-dbf85d3a2045.png)
